core/cpu.c

```diff
@@ -120,11 +120,11 @@ static uint32_t cpu_pop_word(void) {
     return value;
 }
 
-inline static uint8_t cpu_read_in(uint16_t pio) {
+__inline static uint8_t cpu_read_in(uint16_t pio) {
```

Comment:
Use a macro conditional on _MSC_VER instead of this non-portable construct.
Also, "static inline" is a more usual form of "inline static", which I hadn't noticed.